view/frontend/web/template/payment/offline-conveniencestore-form.html

```diff
@@ -20,6 +20,7 @@
                 </div>
             </div>
         </div>
+        <!-- 200 Y as minimum order amount to use convenience store, will be taken in the future from capabilities API-->
```

Comment:
I think it is better not to mention about `200 yen` here as this number might be subjected to change later on and this is a place where we likely to be forgotten to update the most.

Reply:
Also actually this might be out of context of this pull request, however, just a note as this PR is introducing the `restrictedToCurrencies: ['jpy']`.

Currently the form is displaying `an order is too low` warning message only a case of `isActive() && orderValueTooLow()` but we don't have an error message in a case of `!isActive()` (the same as other `restrictedToCurrencies: ['thb']` payment methods that says "Option available only for orders in THB").

Reply:
@guzzilar implemented [here](https://github.com/omise/omise-magento/pull/227)